src/main/io/osd.c

```diff
@@ -825,6 +826,14 @@ static bool osdDrawSingleElement(uint8_t item)
                 break;
             }
 
+#ifdef USE_RC_SMOOTHING_FILTER
+            // Show warning if rc smoothing hasn't initialized the filters
+            if (ARMING_FLAG(ARMED) && !rcSmoothingInitializationComplete()) {
```

Comment:
Should probably check for `osdWarnGetState(OSD_WARNING_RC_SMOOTHING)` (on by default).

Reply:
The only problem with this is that without a settings reset the new warnings are not on by default.  They're only initialized on up the the count so the newly added warnings won't have their bit set.

Reply:
If this goes into 3.4.0, then everybody upgrading from 3.3 will get defaults.

Reply:
@ctzsnooze There's no valid reason for the training to never complete.  It would indicate some kind of problem that we're not handling properly for the protocol.  Since we've already found/fixed cases like this (CRSF and SPI rx) this is to help us catch any others rather than it just silently failing and the user not being aware of the problem (and then not reporting it).  So the warning/beeping will continue as long as the filter initialization hasn't completed.  The idea is that the user won't want to fly this way (and we don't want them to) and will revert to interpolation.

I intend to add logic to "default" to a safe filter setup initially and then update that after the initialization.  But it's more changes than I comfortable making without lots of testing and without forcing another RC.  So I'll add the default logic separately and plan for a 3.4.1 maintenance release.  We (I) need to keep reminding ourselves that rc smoothing is still considered an experimental feature and thus may not work for everybody.  But with feedback we'll improve it.